Ali_Nurisiahsiahi.md

```diff
@@ -0,0 +1,7 @@
+I am Ali Nurisiahsiahi
```

Comment:
English is not my native language and I do lots of typos :) 
I do rely on spellchecker plugins to check everything I type in - if you are using Visual Studio Code - make sure to install [one of the plugins](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker).

I've found [Grammarly](https://app.grammarly.com/) very helpful as well.

Reply:
It's helpful, I will use it.